module/builder/collection/builder_test.go

```diff
@@ -62,9 +51,10 @@ type BuilderSuite struct {
 	chainID      flow.ChainID
 	epochCounter uint64
 
-	headers  storage.Headers
-	payloads storage.ClusterPayloads
-	blocks   storage.Blocks
+	headers          storage.Headers
+	payloads         storage.ClusterPayloads
+	blocks           storage.Blocks
```

Comment:
I would suggest to rename those for extra clarity: 
```suggestion
	clusterHeaders   storage.Headers
	clusterPayloads  storage.ClusterPayloads
	consensusBlocks  storage.Blocks
```
(If I am not mistaken?)